base/sysimg.jl

```diff
@@ -341,6 +341,21 @@ include("docs/Docs.jl")
 using .Docs, .Markdown
 Docs.loaddocs(Core.Inference.CoreDocs.DOCS)
 
+# String constructor docstring, ugly workaround for #16730
+Docs.doc!(Docs.Binding(current_module(),:String), Docs.docstr(Core.svec(
+"""
+    String(v::Vector{UInt8})
+
+Create a new `String` from a vector `v` of bytes containing
+UTF-8 encoded characters.   This function takes "ownership" of
+the array, which means that you should not subsequently modify
+`v` (since strings are supposed to be immutable in Julia) for
+as long as the string exists.
+
+If you need to subsequently modify `v`, use `String(copy(v))` instead.
+"""
+), Dict(:source => :(String(v::Vector{UInt8}) = new(v)), :path=>"boot.jl", :linenumber=>219, :module=>current_module())), Tuple{Vector{UInt8}})
+
```

Comment:
`:path` and `:linenumber` are meant to point to where the docstring is rather than the method that it documents.  Should also just use the call syntax rather than calling `doc!` directly

```julia
"""
...
"""
String(v::Vector{UInt8})
```

to get the correct metadata.

Placing it with the actual definition would be best, but I don't think it would be possible to have it in `boot.jl` since not enough is defined at that point to get the docsystem working. Perhaps put it with some of the later `String` defs, or maybe in `docs/basedocs.jl` where other "difficult" docs have be put.